demo_app_url_for.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, url_for
+
+app = Flask(__name__)
+
+
+@app.route('/')
+def index():
+    """Show index page of the web app.
+
+    And link to the second page.
+    """
+    return f"""
+    <html>
+        <head>
+            <title>Hello world</title>
+        </head>
+        <body>
+            Hello world!
+            <br>
+            <a href='{url_for("second_page", _external=True)}'>'{url_for("second_page")}'</a> to second page.
```

Comment:
Please use the same URL in the link text:
```suggestion
            <a href='{url_for("second_page", _external=True)}'>'{url_for("second_page", _external=True)}'</a> to second page.
```
Or just use “Link” instead of the URL.